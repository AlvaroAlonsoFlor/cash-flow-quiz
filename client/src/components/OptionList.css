label {
    font-size: 1.4em;
    font-variant: small-caps;
}

input[type=radio] label{
    font-variant: small-caps;
}

input[type=radio]:checked~label {
    color: lightskyblue;
}