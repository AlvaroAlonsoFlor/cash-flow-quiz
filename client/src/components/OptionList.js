import React from 'react';
import './OptionList.css'

const OptionList = ({options, handleSelect}) => {

    if (!options) {
        return null
    }

    const list = options.map((option, index) => {
        return(
        <div key={'option' + index}>
            <input type="radio" name='answer' key={'answer' + option.id} id={option.id} value={option.id} onChange={handleSelect}/>
            <label key={option.id}>{option.answer}</label>
            <hr className='radio-separator' />
        </div>
        )       
    })
  
    return(
        <div>{list}</div>     
    )
}

export default OptionList;