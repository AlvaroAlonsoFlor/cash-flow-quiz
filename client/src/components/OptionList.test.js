import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

import OptionList from './OptionList.js'

it('OptionList should render correctly with no props', () => {
    const component = shallow( < OptionList / > );

    expect(component).toMatchSnapshot();
});

