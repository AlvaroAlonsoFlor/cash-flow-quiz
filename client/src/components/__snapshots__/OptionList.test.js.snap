// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptionList should render correctly with no props 1`] = `""`;
